surveys/surveys.go

```diff
@@ -172,6 +186,32 @@ func FindSurveyByName(name string) LauncherSchema {
 	panic("Schema not found")
 }
 
+func GetSupplementaryDataSets(surveyId string, periodId string) ([]DatasetMetadata, error) {
+	datasetList := []DatasetMetadata{}
+	hostURL := settings.Get("SDS_API_URL")
+	log.Printf("SDS Api URL: %s", hostURL)
+	url := fmt.Sprintf("%s/v1/dataset_metadata?survey_id=%s&period_id=%s", hostURL, surveyId, periodId)
+	resp, err := clients.GetHTTPClient().Get(url)
+
+	if err != nil || (resp.StatusCode != 200 && resp.StatusCode != 404){
```

Comment:
At the moment, if the API is down, you can still launch the survey, do we want to prevent that / add a message to say SDS API is not up?.